perun-openapi/openapi.yml

```diff
@@ -389,9 +389,14 @@ components:
           items:
             $ref: '#/components/schemas/AttributePolicyCollection'
         criticalActions:
-          type: array
-          items:
-            $ref: '#/components/schemas/AttributeAction'
+          type: object
+          description: "Represents types of actions (READ, WRITE) that users can perform upon attributes and if they are critical globally or not."
+          additionalProperties:
+            type: boolean
+          example: {
+            READ: false,
+            WRITE: true,
+          }
```

Comment:
Is it possible for the key to still be `AttributeAction` ? This way we lose the type on FE.

Reply:
Unfortunately in our current version of open API, it is probably not possible to define an enum for the key in a map (https://stackoverflow.com/questions/46552863/how-to-write-openapi-3-swagger-specification-for-property-name-in-map-object/46553255#46553255). I'll discuss with others if we can upgrade this version without any conflicts.

Reply:
Definitely not a block for this PR, just a minor inconvenience at worst.